benchmark/scala/src/main/scala/com/microsoft/tpch/ConvertTpchDatasetToParquet.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+// The code is modified based on Wentao's code. Wentao's homepage: https://www.microsoft.com/en-us/research/people/wentwu/
+
+package mypackage
+
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{SaveMode, SparkSession}
+
+object ConvertTpchCsvToParquetApp {
+
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession.builder()
+      // .master(s"local[*]") // UnComment this line to run locally.
```

Comment:
I would remove this since you can pass this as `spark-submit`

Reply:
I see, removed.